source-pendo/source_pendo/api.py

```diff
@@ -0,0 +1,123 @@
+import json
+from logging import Logger
+from typing import AsyncGenerator, Iterable
+
+from estuary_cdk.capture.common import LogCursor
+from estuary_cdk.http import HTTPSession
+
+from .models import Event, EventResponse, Resource, Metadata
+
+API = "https://app.pendo.io/api/v1"
+
+
+async def fetch_resources(
+        http: HTTPSession,
+        entity: str,
+        log: Logger,
+) -> Iterable[Resource]:
+    url = f"{API}/{entity}"
+
+    response = json.loads(await http.request(log, url))
+
+    resources: list[Resource] = []
+    for id, resource in enumerate(response):
+        r = Resource.model_validate_json(json.dumps(resource))
+        r.meta_ = Resource.Meta(op="c", row_id=id)
+        resources.append(r)
+
+    log.info(f"Read {len(resources)} documents from stream {entity}.")
```

Comment:
Not sure if these logs are intended to be kept, but I think it would be best to remove them - this information is available from stats actually

Reply:
I'll make sure to remove them after addressing all comments. They're helpful for me when stats are visible locally.